src/shared/components/mutationTable/column/DiscreteCNAColumnFormatter.tsx

```diff
@@ -18,15 +18,186 @@ export default class DiscreteCNAColumnFormatter {
+            facetsCall = "Amp";
+        } else if (wgd === "no WGD" && mcn === 6 && lcn === 3) {
+            facetsCall = "Amp";
+        } else if (wgd === "WGD" && mcn === 0 && lcn === 0) {
+            facetsCall = "HOMDEL";
+        } else if (wgd === "WGD" && mcn === 1 && lcn === 0) {
+            facetsCall = "Loss Before & After";
+        } else if (wgd === "WGD" && mcn === 2 && lcn === 0) {
+            facetsCall = "Loss Before";
+        } else if (wgd === "WGD" && mcn === 3 && lcn === 0) {
+            facetsCall = "CNLOH Before & Loss";
+        } else if (wgd === "WGD" && mcn === 4 && lcn === 0) {
+            facetsCall = "CNLOH Before";
+        } else if (wgd === "WGD" && mcn === 5 && lcn === 0) {
+            facetsCall = "CNLOH Before & Gain";
+        } else if (wgd === "WGD" && mcn === 6 && lcn === 0) {
+            facetsCall = "Amp (LOH)";
+        } else if (wgd === "WGD" && mcn === 1 && lcn === 1) {
+            facetsCall = "Double Loss After";
+        } else if (wgd === "WGD" && mcn === 2 && lcn === 1) {
+            facetsCall = "Loss After";
+        } else if (wgd === "WGD" && mcn === 3 && lcn === 1) {
+            facetsCall = "CNLOH After";
+        } else if (wgd === "WGD" && mcn === 4 && lcn === 1) {
+            facetsCall = "Loss & Gain";
+        } else if (wgd === "WGD" && mcn === 5 && lcn === 1) {
+            facetsCall = "Amp";
+        } else if (wgd === "WGD" && mcn === 6 && lcn === 1) {
+            facetsCall = "Amp";
+        } else if (wgd === "WGD" && mcn === 2 && lcn === 2) {
+            facetsCall = "Tetraploid";
+        } else if (wgd === "WGD" && mcn === 3 && lcn === 2) {
+            facetsCall = "Gain";
+        } else if (wgd === "WGD" && mcn === 4 && lcn === 2) {
+            facetsCall = "Amp";
+        } else if (wgd === "WGD" && mcn === 5 && lcn === 2) {
+            facetsCall = "Amp";
+        } else if (wgd === "WGD" && mcn === 6 && lcn === 2) {
+            facetsCall = "Amp";
+        } else if (wgd === "WGD" && mcn === 3 && lcn === 3) {
+            facetsCall = "Amp (Balanced)";
+        } else if (wgd === "WGD" && mcn === 4 && lcn === 3) {
+            facetsCall = "Amp";
+        } else if (wgd === "WGD" && mcn === 5 && lcn === 3) {
+            facetsCall = "Amp";
+        } else if (wgd === "WGD" && mcn === 6 && lcn === 3) {
+            facetsCall = "Amp";
+        } else {
+            facetsCall = "NA";
+        }
+        return facetsCall;
+    }
+   
+    public static formatFacetsCNAData(facetsCNAData:string) {
+        let color = "";
+        let size = "";
+        if (facetsCNAData.includes("Amp")) {
+            color = "red";
+            size = "12px";
+        } else if (facetsCNAData.includes("Gain")) {
+            color = "red";
+            size = "smaller"; 
+        } else if (facetsCNAData.includes("Loss")) {
+            color = "blue";
+            size = "smaller";
+        } else if (facetsCNAData.includes("Homdel")) {
```

Comment:
there's both a "HOMDEL" and "Homdel". If they are truly expected to be the same then we should normalize the facets call value to "Homdel" so they get formatted the same way